PODMAN.md

```diff
@@ -3,7 +3,7 @@
 Install podman 3.1+, then run it in docker compatible mode:
```

Comment:
I don't think this makes any sense anymore right?

Reply:
Are you speaking about the version or the docker compatible mode?

Reply:
I was about to say both but 3.1 is barely a year old so that should likely stay.

The docker compatible mode doesn't make any sense to me tbh.

Reply:
From what I've heard, it probably worked with 3.3 at some point. On 3.4 it is not working for me.

I think running in docker compatible mode means letting podman listen on a socket, which it doesn't do by default. Mentioning this probably still makes sense.

Reply:
I'm pretty sure it does by default now on Fedora >=35,

Reply:
You don't need to start podman.service or podman.socket to use podman. In contrast to docker it works daemonless by default.

Reply:
Yeah apologies that's a fair point, I think my reliance on systemd units to do this isn't the main approach people will take here.

Reply:
Nothing to apologize for :) I think systemd units are actually the way to easily run podman in docker compatibility mode.